e2e/libs/node/node.py

```diff
@@ -38,7 +38,7 @@ def reboot_node(self, running_on_node_name, reboot_node_name, shut_down_time_in_
             mapping = yaml.safe_load(f)
             reboot_node_id = mapping[reboot_node_name]
 
-        filepath = './litmus/reboot-node.yaml'
+        filepath = './templates/litmus/reboot-node.yaml'
```

Comment:
Suggest having:
- a template util function to get the file
- use const variable for any constant file path like `tmp/instance_mapping`

You can do it later over time.

Reply:
Created https://github.com/longhorn/longhorn-tests/pull/1530#discussion_r1328727134 for this.

Reply:
@yangchiu which issue you have been created?

Reply:
Sorry for the wrong link.. Updated!